src/Components/Publishing/Fixtures/Components.ts

```diff
@@ -176,6 +171,25 @@ export const UnitPanel = {
   },
 }
 
+export const Display = type => ({
+  ...Campaign,
+  canvas: (() => {
+    switch (type) {
+      case "overlay":
```

Comment:
This model has changed a little bit since building the UI--- there is no `type` field, only `layout` (3 total, image and video are the same)

- Overlay
- Standard 
- Slideshow

Reply:
This is for creating stories for each of the media types—kinda confusing b/c of the `type` field before so I renamed it to `mediaType`, but I think this makes sense.

![image](https://user-images.githubusercontent.com/555859/31559537-f5ca53ca-b01e-11e7-8481-22965be42e15.png)